src/main/java/org/example/C03_08_OddNumbersAndSum.java

```diff
@@ -0,0 +1,24 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C03_08_OddNumbersAndSum {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter the input number of terms : ");
+        int input = scanner.nextInt();
+
+        System.out.println("The first " + input + " odd natural numbers are:");
+        int sum = 0;
+        for (int i = 1; i <= input; i++) {
+            int oddNumbers = 2 * i - 1;
+            System.out.print(oddNumbers + " ");
```

Comment:
Check if the number of odd numbers printed equals the input number.

Reply:
done. it is ok